Sources/Swift/SentryExperimentalOptions.swift

```diff
@@ -1,9 +1,22 @@
 @objcMembers
 public class SentryExperimentalOptions: NSObject {
+    /**
+     * Enables swizzling of`NSData` to automatically track file operations.
+     *
+     * - Note: Swizzling is enabled by setting ``SentryOptions.enableSwizzling`` to `true`.
+     *         This option allows you to disable swizzling for `NSData` only, while keeping swizzling enabled for other classes.
+     *         This is useful if you want to use manual tracing for file operations.
+     */
+    public var enableDataSwizzling = true
+
     /**
      * Enables swizzling of`NSFileManager` to automatically track file operations.
+     *
+     * - Note: Swizzling is enabled by setting ``SentryOptions.enableSwizzling`` to `true`.
+     *         This option allows you to disable swizzling for `NSFileManager` only, while keeping swizzling enabled for other classes.
+     *         This is useful if you want to use manual tracing for file operations.
      */
-    public var enableFileManagerSwizzling = false
+    public var enableFileManagerSwizzling = true
```

Comment:
`h`: Why did you decide to turn this experimental feature on by default? Do we have enough data to know that the swizzling of the FileManager doesn't cause any major problems? Am I missing something?

Side note: We can safely set `enableDataSwizzling` to true` because we swizzle NSData already for years without major problems. It's acceptable to deviate the default values here.

Reply:
I'll revert and create an issue so we eventually enable it.